kpi/views/v2/service_usage.py

```diff
@@ -1,16 +1,22 @@
 # coding: utf-8
+from django.utils.decorators import method_decorator
+from django.views.decorators.cache import cache_page
+from django.views.decorators.vary import vary_on_cookie
 from rest_framework import (
     renderers,
     viewsets,
 )
-from rest_framework.decorators import action
 from rest_framework.response import Response
 
 from kpi.permissions import IsAuthenticated
 from kpi.serializers.v2.service_usage import ServiceUsageSerializer
 from kpi.utils.object_permission import get_database_user
 
 
+@method_decorator(cache_page(60 * 30), name='retrieve')
+@method_decorator(cache_page(60 * 30), name='list')
```

Comment:
Let's use a setting for this (which could even be configurable with env variable, then fallback on your default). We could have a shorter timeout in dev settings than prod. 

Maybe 30 minutes is a little bit too long, what about reverting it to 15?

Reply:
I've extracted this out to `ENDPOINT_CACHE_DURATION`. This setting is used by `/service_usage/` and `/stripe/products/`, but that can be refactored if we later need to set those cache settings independently.

The only thing I'm not sure about is using a shorter timeout in dev settings; I'm worried that could make it easier to miss unwanted results of caching while debugging.